internal/config/config.go

```diff
@@ -28,31 +28,26 @@ const EnvPrefix string = "RELAYER"
 type NeutronChainConfig struct {
 	RPCAddr        string        `required:"true" split_words:"true"`
 	RESTAddr       string        `required:"true" split_words:"true"`
-	ChainID        string        `required:"true" split_words:"true"`
 	HomeDir        string        `required:"true" split_words:"true"`
 	SignKeyName    string        `required:"true" split_words:"true"`
-	Timeout        time.Duration `required:"true" split_words:"true"`
+	Timeout        time.Duration `required:"true" split_words:"true" default:"10s"`
```

Comment:
:thinking: why is it `required:"true"` if there is a default specified?

Reply:
I think it does not matter much? if value is not present it won't fail anyway.

Reply:
That's really strange but ok.